src/Cms/App.php

```diff
@@ -806,7 +806,7 @@ public function kirbytext(string $text = null, array $data = [], bool $inline =
 
         $text = $this->apply('kirbytext:before', compact('text'), 'text');
         $text = $this->kirbytags($text, $options);
-        $text = $this->markdown($text, $options['markdown']);
+        $text = $this->markdown($text, $options['markdown'], $options['markdown']['inline']);
```

Comment:
The `markdown()` method has no 3rd parameter and the `inline` parameter is already processed in `$options`. Having an issue?

Reply:
Thanks for pointing that out. I put my code in the wrong spot.